docs/deploy-and-review-process.md

```diff
@@ -0,0 +1,59 @@
+# Deploy and review process
+
+This documentation outlines our current processes for how we deploy the latest changes to ethereum.org, and how we prioritize items in our review process.
+
+## Deploy process
+
+Anything merged into the `dev` branch of [ethereum.org](http://ethereum.org) will go live on ethereum.org at the next release. Currently, we deploy the latest changes to [ethereum.org](http://ethereum.org) twice per week.
```

Comment:
Just a thought for an additional sentence here, perhaps at the beginning:

> Our `dev` branch acts as a staging branch, where all PRs are merged into first, allowing checks for unexpected behavior before they go live.

Reply:
And is it worth mentioning the `master` branch here as the canonical live branch?

Reply:
```suggestion
Anything merged into the `dev` branch of [ethereum.org](http://ethereum.org) will go live on ethereum.org at the next release. Currently, we deploy the latest changes to [ethereum.org](http://ethereum.org) twice per week. When deploying, we merge the `dev` branch into the `master` branch. The `master` branch is the latest live version of [ethereum.org](https://ethereum.org).
```